### PR TITLE
Allow overriding of NAMED_ARGUMENTS

### PR DESCRIPTION
**Pull request by derekbekoe**
Closes https://github.com/Microsoft/knack/issues/11

### CODE REVIEW

File: knack/parser.py [hunk 1]
```diff
@@ -10,6 +10,22 @@
...
+    'dest',
+    'nargs',
+    'const',
+    'default',
+    'type',
+    'choices',
+    'required',
+    'help',
+    'metavar',
+]
```
Comment (derekbekoe) [inline knack/parser.py]: In your 'KnackConversion' branch, [you also included 'action'](https://github.com/Azure/azure-cli/blob/KnackConversion/src/azure-cli-core/azure/cli/core/parser.py#L21) but I haven't included it in the PR to Knack as I didn't see it in argparse in https://github.com/python/cpython/blob/master/Lib/argparse.py#L748.

Comment (derekbekoe) [reply knack/parser.py]: Is this good or is there a scenario where 'action' should be added?

Comment (tjprescott) [reply knack/parser.py]: Action is definitely supported by argparse and thus should be in the list.

Comment (tjprescott) [reply knack/parser.py]: `action='store_true'`  is an example.

**Comment by derekbekoe** [comment]
@tjprescott I've added 'action' back in

**Pull request merged**